src/library/scala/beans/ScalaBeanInfo.scala

```diff
@@ -32,15 +31,9 @@ abstract class ScalaBeanInfo(clazz: java.lang.Class[_],
   override def getPropertyDescriptors() = pd
   override def getMethodDescriptors() = md
 
-  // override def getAdditionalBeanInfo() = Array(Introspector getBeanInfo clazz.getSuperclass)
-
   private def init() {
-    var i = 0
```

Comment:
My guess is that this was originally done for performance reasons, but I think it's fine to replace it with a foreach loop over a range.